source/api/commands/rightclick.md

```diff
@@ -0,0 +1,199 @@
+---
+title: rightclick
+---
+
+Right click a DOM element.
+
+{% note warning %}
+`.rightclick()` will not open context menus native to the browser. `.rightclick()` should be used to test the behavior of an application where a user would right click a mouse.
+{% endnote %}
+
+# Syntax
+
+```javascript
+.rightclick()
+.rightclick(options)
+.rightclick(position)
+.rightclick(position, options)
+.rightclick(x, y)
+.rightclick(x, y, options)
+```
+
+## Usage
+
+**{% fa fa-check-circle green %} Correct Usage**
+
+```javascript
+cy.get('.menu').rightclick()       // Right click on .menu
+cy.focused().rightclick()          // Right click on el with focus
+cy.contains('Today').rightclick()  // Right click on first el containing 'Today'
+```
+
+**{% fa fa-exclamation-triangle red %} Incorrect Usage**
+
+```javascript
+cy.rightclick('button')    // Errors, cannot be chained off 'cy'
+cy.window().rightclick()   // Errors, 'window' does not yield DOM element
+```
+
+## Arguments
+
+**{% fa fa-angle-right %} position** ***(String)***
+
+The position where the right click should be issued. The `center` position is the default position. Valid positions are `topLeft`, `top`, `topRight`, `left`, `center`, `right`, `bottomLeft`, `bottom`, and `bottomRight`.
+
+{% imgTag "/img/api/coordinates-diagram.jpg" "cypress-command-positions-diagram" %}
+
+**{% fa fa-angle-right %} x** ***(Number)***
+
+The distance in pixels from the element's left to issue the right click.
+
+**{% fa fa-angle-right %} y** ***(Number)***
+
+The distance in pixels from the element's top to issue the right click.
+
+**{% fa fa-angle-right %} options** ***(Object)***
+
+Pass in an options object to change the default behavior of `.rightclick()`.
+
+Option | Default | Description
+--- | --- | ---
+`log` | `true` | {% usage_options log %}
+`force` | `false` | {% usage_options force rightclick %}
+`multiple` | `false` | {% usage_options multiple rightclick %}
+`timeout` | {% url `defaultCommandTimeout` configuration#Timeouts %} | {% usage_options timeout .rightclick %}
+
+## Yields {% helper_icon yields %}
+
+{% yields same_subject .rightclick %}
+
+# Examples
+
+## No Args
+
+### Right click the menu
+
+```javascript
+cy.get('#context-menu').rightclick()
```

Comment:
usually a context-menu is spawned as a result of a right click, so this example might be confusing (I guess this was in the driver tests?)